homeassistant/components/moon/__init__.py

```diff
@@ -1,9 +1,28 @@
 """The Moon integration."""
+import logging
+
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 
 from .const import PLATFORMS
 
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
```

Comment:
I think we can just update the entry to "remove" the title in the normal setup (using `hass.config_entries.async_update_entry()`) and don't need to bump the version and implement migration.